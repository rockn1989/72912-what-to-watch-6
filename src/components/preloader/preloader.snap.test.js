import React from 'react';
import {render} from '@testing-library/react';
import Preloader from './preloader';

it(`Should Preloader render correctly`, () => {
  const {container} = render(<Preloader />);
  expect(container).toMatchSnapshot();
});
