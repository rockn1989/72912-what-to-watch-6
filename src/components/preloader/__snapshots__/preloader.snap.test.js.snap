// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Preloader render correctly 1`] = `
<div>
  <div
    class="preloader"
    style="margin: 30px auto;"
  >
    <p
      style="font-size: 24px; line-height: 32px; text-align: center; font-weight: bold;"
    >
      Loading...
    </p>
  </div>
</div>
`;
